src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/CreateAdjudicationRequest.kt

```diff
@@ -76,7 +76,18 @@ data class IncidentToCreate(
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class RepairToCreate(
-  @Schema(description = "Type of repairs", example = "PLUM")
+  @Schema(
```

Comment:
This will have a small impact in the update service, since it will now accept and enum, which is a good thing